third-party/llvm/llvm-src/tools/clang/utils/check_cfc/setup.py

```diff
@@ -2,7 +2,8 @@
     python.exe setup.py py2exe
```

Comment:
This file and `update_cc_test_checks.py` are a bundled copy of llvm/clang, so I don't think we want to modify them, but instead wait for upstream to have the fix (note that our bundled version gets stripped out of the Homebrew Formulae and we use a system version from homebrew.)